doc/devel/coding_guide.rst

```diff
@@ -165,6 +165,48 @@ C/C++ extensions
 
 
 
+PR Review guidelines
+====================
+
+* We run on a 'use your judgement' system (for better or worse) and do
+  not have a strict nose-count rule.  If you have a commit bit, then
```

Comment:
It is somewhat counter intuitive that going to a '2 devs must sign off' scheme would increase the rate at which things get merged, but I suspect it might be true.